celery/extractions.py

```diff
@@ -14,13 +20,109 @@
 FONCIER_EXTRACTS_DIR = env.get('FONCIER_EXTRACTS_DIR', '/tmp')
 FONCIER_STATIC_DIR = env.get('FONCIER_STATIC_DIR')
 
+PG_CONNECT_STRING = env.get("PG_CONNECT_STRING")
+
 taskmanager = Celery('extractions',
                      broker=CELERY_BROKER_URL,
                      backend=CELERY_RESULT_BACKEND)
 
+
+def run_command(args):
+    """
+    Run command specified by args. If exit code is not 0 then full command line, STDOUT, STDERR are printed and an
+    Exception is raised
+    :param args: array of argument
+    :return: None
+    """
+    p = Popen(args, stdout=PIPE, stderr=PIPE)
+    p.wait()
```

Comment:
I'm wondering if we should specify a timeout argument to wait().
https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait